epictrack-web/src/services/rtkQuery/projectInsights.tsx

```diff
@@ -18,8 +18,8 @@ export const projectInsightsApi = createApi({
     prepareHeaders,
   }),
   endpoints: (builder) => ({
-    getProjects: builder.query<Project[], void>({
-      query: () => `projects`,
+    getProjects: builder.query<Project[], boolean | void>({
+      query: (is_active = true) => `projects?is_active=${is_active}`,
```

Comment:
Can you check the backend if you have this is_active query string is taken care of

Reply:
thanks for the heads up, I added some handling